rpm/0007-Revert-deprecating-shell-3-5196.patch

```diff
@@ -0,0 +1,62 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Daniel Suni <daniel.suni@jolla.com>
+Date: Fri, 17 Feb 2023 11:48:56 +0200
+Subject: [PATCH] Revert "deprecating --shell (3) (#5196)"
+
+This reverts commit 7ad735deafa80114a17b20790de63f7e973b1bb4.
+
+This commit makes firejail attempt to use /bin/bash to launch every process
+that uses a "--" argument. Sailjail uses this, but we do *not* want shell
+execution, since it will fail.
+
+There are a number of bug reports in firejail upstream with regards to this
+very odd feature - see e.g. https://github.com/netblue30/firejail/issues/5659
```

Comment:
From the bug report it seems to me that specifying binaries by their full path should work still so I'm wondering do we really need this revert patch. AFAIR sailjail should always use full paths.

Reply:
Full paths don't seem to make a difference. Both looking a the code as well as testing on the device shows that as long as you have a "--" parameter (which sailjail specifically adds) firejail will try to run the program using the (nonexistent) shell unless the patch is reverted.

Reply:
Ok.